### DISCUSSION TITLE
How can I downgrade the phalcon version?

### DISCUSSION THREAD

**Discussion by jasmineNik** [phalcon/cphalcon#15910] [Q&A]
I've installed the phalcon by following the docs. Then I tried downgrade the version by checking out to the 3.3.x branch and install again but I still have an error when I'm trying to do composer install.
Here is the error:
`dmkit/phalcon-jwt-auth 1.0.0 requires ext-phalcon ^3.0 -> it has the wrong version installed (4.1.2)`

So can anybody help me to install the correct version?
My php version is 7.4 /The project requires php >=7.1/.

**Comment by niden** [comment]
Uninstall phalcon. Ensure that you also delete the `phalcon.so` module from your server. Check out the branch you need and then compile from sources. That should do it.

**Reply by jasmineNik** [reply]
I did it. I have checked out to branch 3.3.x  but it didn't help me. Still the version 4.1.2
